text-ui-test/input.txt

```diff
@@ -0,0 +1,9 @@
+todo borrow book
```

Comment:
Consider adding testing for erroneous input. We ideally want to handle errors effectively throughout our dev progress.